Mathlib/Data/Set/Card.lean

```diff
@@ -168,7 +168,7 @@ theorem encard_le_coe_iff {k : ℕ} : s.encard ≤ k ↔ s.Finite ∧ ∃ (n₀
     fun ⟨_,⟨n₀,hs, hle⟩⟩ ↦ by rwa [hs, Nat.cast_le]⟩
 
 @[simp]
-theorem encard_prod : (s ×ˢ t).encard = s.encard * t.encard := by
+theorem encard_prod {s : Set α} {t : Set β} : (s ×ˢ t).encard = s.encard * t.encard := by
   unfold encard
```

Comment:
note to reviewer: it doesn't work to try to combine this unfold with the following simp--it causes simp to loop.